src/modules/aws_cloudfront/index.ts

```diff
@@ -117,17 +118,30 @@ class DistributionMapper extends MapperBase<Distribution> {
       WebACLId: e.webACLId,
       Origins: { Quantity: e.origins.length, Items: e.origins },
       PriceClass: distributionConfig.DistributionConfig?.PriceClass,
-      Aliases: distributionConfig.DistributionConfig?.Aliases,
+      Aliases: { Quantity: e.alternateDomainNames?.length ?? 0, Items: e.alternateDomainNames ?? [] },
```

Comment:
why do we track Quantity here? is not just length of items?

Reply:
It is. I'm not sure if I follow, since it's being evaluated dynamically.

Reply:
ah ok ... that explain, if that's mandatory from AWS... looked as strange parameter to me